gtsfm/runner/run_scene_optimizer.py

```diff
@@ -42,5 +44,16 @@ def run_scene_optimizer() -> None:
 
 
 if __name__ == "__main__":
-    run_scene_optimizer()
-
+    parser = argparse.ArgumentParser(description="GTSFM with intrinsics and image names stored in COLMAP-format")
+    parser.add_argument(
+        "--dataset_root", type=str, default=os.path.join(DATA_ROOT, "set1_lund_door"), help=""
+    )
+    parser.add_argument(
+        "--image_extension", type=str, default="JPG", help=""
+    )
+    parser.add_argument(
+        "--max_frame_lookahead", type=int, default=20, help="maximum number of consecutively ordered frames to consider for matching/co-visibility"
```

Comment:
“Maximum number of consecutive frames” is probably simpler. Are we losing some meaning then?

Reply:
thanks -- updated